tor_ocr/main.py

```diff
@@ -94,32 +130,18 @@ def run(config):
     )
     image_post = config.r.submission(id=clean_id(new_post))
 
-    # download image for processing
-    # noinspection PyUnresolvedReferences
-    try:
-        filename = wget.download(image_post.url)
-    except (
-            urllib.error.HTTPError,
-            urllib.error.URLError
-    ):
-        # what if the post has been deleted or we don't know what it is?
-        # Ignore it and continue.
-        return
+    url = image_post.url
```

Comment:
Seeing this here now, couldn't we just use `image_post.url` directly as the arg for `process_image()`? If we were doing transformations or validations, it might make sense to copy it and pull it out, but I don't see us needing to do so at the moment.

Reply:
Good idea, I was just using the original parameters from before, but that makes more sense.